.github/CODEOWNERS

```diff
@@ -30,7 +30,7 @@ subprojects/enterprise-workers/             @gradle/ge-build-insights @gradle/bt
 
 # JVM platform
 platforms/jvm/                              @gradle/bt-jvm
-subprojects/code-quality/                   @gradle/bt-jvm
+platforms/code-quality/                     @gradle/bt-jvm
```

Comment:
`platforms/code-quality` does not exist

The original `subprojects/code-quality` line can just be deleted entirely since the `platforms/jvm` line above contains the `platforms/jvm/code-quality` directory.